src/cms/models/push_notifications/push_notification_channel.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+
+class PushNotificationChannel(models.Model):
+    """
+    Data model representing a push notification channels
+
+    :param channel: The channel of the push notification
+    """
+
+    channel = models.CharField(max_length=60)
```

Comment:
```suggestion
    name = models.CharField(max_length=60)
```
Since the model is already the channel, I think it's inconsistent to have also a `channel` attribute...
This would result in e.g. `push_notification.channel.channel` instead of `push_notifications.channel.name` etc...<
But that's just my preference...

Reply:
Valid point.